src/java/org/apache/cassandra/config/EncryptionOptions.java

```diff
@@ -67,6 +68,42 @@ public String description()
         }
     }
 
+    public enum ClientAuth
+    {
+        REQUIRED("true"),
+        NOT_REQUIRED("false"),
+        OPTIONAL("optional");
+        private final String value;
+        private static Map<String, ClientAuth> values = new HashMap<>();
```

Comment:
I would prefer a case insensitive treemap here, given that we are dealing with user input, so both "Required" and "REQUIRED" would be valid configuration options. Also, I think it is preferred to use final and I would uppercase this field to denote it is a static final field
```suggestion
        private static final Map<String, ClientAuth> VALUES = new TreeMap<>(String.CASE_INSENSITIVE_ORDER);
```